README.md

```diff
@@ -1,6 +1,6 @@
 # Quick Start
 
-TODO: Description
+A script that uses sphinx to develop documentation for ApeWorX plugins.
```

Comment:
refactor idea: We can make this less of a script and more of a package / sphinx plugin (at some point).

Something like:

```python
from ape_plugins_docs import build_docs
```

(btw not saying we really need to publish this to pypi, only talking about the usage)